barretenberg/cpp/src/barretenberg/relations/databus_lookup_relation.hpp

```diff
@@ -184,16 +185,18 @@ template <typename FF_> class DatabusLookupRelationImpl {
     template <typename Accumulator, size_t bus_idx, typename AllEntities, typename Parameters>
     static Accumulator compute_write_term(const AllEntities& in, const Parameters& params)
     {
+        using MonomialAccumulator = typename Accumulator::MonomialAccumulator;
         using View = typename Accumulator::View;
```

Comment:
View and ParameterView seem to be not used anymore?

Reply:
fixed